apps/web/app/features/page/page.tsx

```diff
@@ -24,18 +24,20 @@ interface PageHeadingProps {
   subtitle?: ReactNode;
   children?: ReactNode;
 }
-export function PageHeading({ title, subtitle = '', children }: PageHeadingProps) {
+export function PageHeading({ title, subtitle, children }: PageHeadingProps) {
   return (
     <Flex my="space.07" flexDir={['column', 'column', 'row']} gap={[null, null, 'space.08']}>
-      <Box width="50%">
+      <Box flex={1}>
         <Page.Title textStyle="heading.02" maxW="400px">
           {title}
         </Page.Title>
       </Box>
-      <Box width="50%">
-        <Page.Subtitle mt={['space.03', 'space.03', 0]} style={{ whiteSpace: 'pre-line' }}>
-          {subtitle}
-        </Page.Subtitle>
+      <Box flex={1}>
+        {subtitle && (
+          <Page.Subtitle mt={['space.03', 'space.03', 0]} whiteSpace="pre-line">
```

Comment:
Not related to your work but in the past we moved the `extension` away from using array syntax for breakpoints towards more specific named ones like: ` mt={{ base: 'space.04', sm: 'space.07' }}`. 

Although more verbose I think that can be more easy to read and decipher than array sytax

Reply:
I love this syntax, it's super simple to understand and read imo!?